meeting_plans.md

```diff
@@ -0,0 +1,48 @@
+# Meeting plans
+
+## Meeting frequency...
+
+* We can start by trying to schedule meetups once every two months
+  * We start conservative, as we don't want to “burn” the community, neither suffer from poor presentations quality
+  * If filling the slots becomes easy or difficult, we will revisit the schedule
+
+## ...starting on
+
+October 2019
+
+## Meetups shape
+
+* Between 1h and 1h:30 min
+* 1, 2 or 3 sessions, each of <=45m (not a hard limit - exceptions allowed if justified)
+* Normally in English...
+  * ...but specific dates where we can consolidate talks from speakers who are not comfortable presenting
+  in English (the “local”/non-English day?)
+
+## Meetups topics
+
+If there is enough proposals, we can try to introduce "topic days". Examples:
+
+* Local/Non-English day
+* Culture day: let's speak about non-tech topics around CD
+* Papers day: lets present research papers touching CI/CD
+* Conference day: let's summarize/play/reproduce/analyse talks from a conference we find relevant
+* Tech day: lets just speak about tech
+* Vendor day: lets allow one (or multiple) vendors to present their proposal/services/tools to implement CI/CD
+* Tech-battle day: let's benchmark/compare and discuss tooling/platform
+* Workshop day: Lets play with the tool X together
```

Comment:
Can I suggest `hands-on` day?

Reply:
You mean a specific proposal, or you prefer `s/hands-on/Workshop/` ?